wandb/sdk/interface/artifacts.py

```diff
@@ -727,11 +727,36 @@ def get_artifact(self, artifact_id):
     def store_artifact(self, artifact):
         self._artifacts_by_id[artifact.id] = artifact
 
+    def cleanup(self, target_size: int) -> int:
```

Comment:
I think this function will leave the directory structure untouched - do we want to potentially cleanup any emptied directories?

Reply:
I considered it, but figured they don't really take up any space so I didn't really see a pressing need to. No one should be poking around in the cache anyway :)